src/lib/transaction/Transaction.svelte

```diff
@@ -53,7 +54,7 @@
 
 {#if $isValidXdr}
     <div class="simple-signer payment-tx">
-        {#if xBull || privateKey}
+        {#if xBull || privateKey || freighter}
```

Comment:
If this will keep growing it'll be better to create a variable up top called isWalletConnected or something around those lines?